ci-operator/jobs/openshift/route-monitor-operator/openshift-route-monitor-operator-master-presubmits.yaml

```diff
@@ -196,22 +193,12 @@ presubmits:
           readOnly: true
       serviceAccountName: ci-operator
       volumes:
-      - name: boskos
-        secret:
-          items:
-          - key: credentials
-            path: credentials
-          secretName: boskos-credentials
       - name: ci-pull-credentials
         secret:
           secretName: ci-pull-credentials
-      - name: cluster-profile
-        projected:
-          sources:
-          - secret:
-              name: cluster-secrets-gcp-openshift-gce-devel-ci-2
-          - configMap:
-              name: cluster-profile-gcp-openshift-gce-devel-ci-2
+      - name: hive-hive-credentials
```

Comment:
What's the purpose of these?

Reply:
To pass the kubeconfig to the script if I'm right

Most of these are generated so I just worry about ci-operator/jobs/openshift/route-monitor-operator/openshift-route-monitor-operator-master-presubmits.yaml